clippy_lints/src/casts/mod.rs

```diff
@@ -80,7 +80,8 @@ declare_clippy_lint! {
     /// ### What it does
     /// Checks for casts between numerical types that may
     /// truncate large values. This is expected behavior, so the cast is `Allow` by
-    /// default.
+    /// default. It suggests user either explicitly ignore the lint,
+    /// or use `try_from()` and handle the truncation, default, or panic explicitly.
```

Comment:
Could you also add a `Use instead:` block below, like this:

````md
Use instead:
```
fn as_u8(x: u64) -> u8 {
    if let Ok(x) = u8::try_from(x) {
        x
    } else {
        todo!();
    }
}
// Or
fn as_u16(x: u64) -> u16 {
    #[allow(clippy::cast_possible_truncation)]
    x as u16
}
```
````